core/common/src/ChangedElements.ts

```diff
@@ -11,26 +11,57 @@ import { AxisAlignedBox3dProps } from "./geometry/Placement";
  * @module Entities
  */
 
-/** @internal */
+/** Type of change bitflags that show what changed on an element
```

Comment:
Can you link to related APIs? e.g.,
`Bitflags describing which aspects of an [Element]($backend) changed as part of a [[ChangedElements]].`
`The [[Placement]] of a [GeometricElement]($backend) changed.`
etc.